src/index.html

```diff
@@ -3,12 +3,32 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="#">
+          <img src="/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="#" class="is-active">Apple</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="is-active">Apple</a>
          </li>
```
Consider fixing indents according to the code style guide